aiidalab/utils.py

```diff
@@ -51,9 +51,13 @@ def load_app_registry():
             return json.loads(file.read())
     else:
         try:
-            return requests.get(AIIDALAB_REGISTRY).json()
-        except ValueError:
+            registry = requests.get(AIIDALAB_REGISTRY)
+            # Now check also the exit code, only accept a 200 (OK, request succeeded)
+            assert registry.status_code == 200
```

Comment:
```suggestion
            registry.raise_for_status()
```

Reply:
Actually, I'm not 100% sure that the way it's coded right now it would ever raise the intercepted exception, so this is actually more of a change request.